crates/bitwarden-exporters/src/cxf/editable_field.rs

```diff
@@ -0,0 +1,225 @@
+use bitwarden_vault::FieldType;
+use credential_exchange_format::{
+    EditableField, EditableFieldBoolean, EditableFieldConcealedString, EditableFieldString,
+    EditableFieldWifiNetworkSecurityType,
+};
+
+use crate::Field;
+
+/// Helper function to create a Field from any EditableField type
+pub(super) fn create_field<T>(name: String, field: &T) -> Field
+where
+    T: EditableFieldToField,
+{
+    Field {
+        name: Some(name),
+        value: Some(field.field_value()),
+        r#type: field.field_type() as u8,
+        linked_id: None,
+    }
+}
+
+/// Trait to convert EditableField types to Field values and types
```

Comment:
We should probably be a bit more clear in who owns each type, as they have very similar names and they're easy to confuse.
```suggestion
/// Trait to convert CXP EditableField types to Bitwarden Field values and types
```